internal/config/config.yaml

```diff
@@ -0,0 +1,5 @@
+server:
+  customerPort: 3002
+  port: 3001
```

Comment:
that's the same port of the payment service

Reply:
done